client/app/bundles/course/container/Sidebar/PinSidebarButton.tsx

```diff
@@ -0,0 +1,46 @@
+import { defineMessages } from 'react-intl';
+import { ChevronLeft, ChevronRight } from '@mui/icons-material';
+import { Typography } from '@mui/material';
+
+import useTranslation from 'lib/hooks/useTranslation';
+
+const translations = defineMessages({
+  minimiseSidebar: {
```

Comment:
I originally thought of using the word _unpin_, too, but I thought it might be confusing, given that most people will load Coursemology with the sidebar pinned _first_. _Unpin_ may make more sense if _Pin_ action was already done, but it's almost never the case here.